ios/Classes/RSIShareViewController.swift

```diff
@@ -67,6 +67,15 @@ open class RSIShareViewController: SLComposeServiceViewController {
                                                          index: index,
                                                          content: content)
                                     }
+                                case .pkpass:
+                                    if let raw = data as? NSData{
```

Comment:
personally I would prefer to have it in brackets
```suggestion
                                    if (let raw = data as? NSData) {
```
or at least a space before the curly bracket?